pkg/models/user.go

```diff
@@ -241,6 +250,13 @@ type UserSearchHitDTO struct {
 	AuthModule    AuthModuleConversion `json:"-"`
 }
 
+type UserDisplayDTO struct {
+	Id    int64  `json:"id,omitempty"`
+	Name  string `json:"name,omitempty"`
+	Login string `json:"login,omitempty"`
+	Email string `json:"email,omitempty"`
```

Comment:
Is email is safe to expose as part of event to other users? Asking since it's personal info and I just triggered.